lib/helper/Puppeteer.js

```diff
@@ -409,6 +410,15 @@ class Puppeteer extends Helper {
     }
   }
 
+  async _addErrorListener(page) {
+    if (!page) {
+      return;
+    }
+    page.on('error', async (error) => {
+      console.error('Puppeteer page error', error);
```

Comment:
Can you use this.debug instead console.log?

Reply:
I think console.error is better here as this event:

> Emitted when the page crashes.

So even in non-debug mode we should see that this was an internal crash of browser/puppeteer and not the application itself or a test